src/lib/yang/parser.lua

```diff
@@ -189,7 +204,19 @@ end
 function Parser:parse_string()
    if self:check("'") then return self:parse_qstring("'")
    elseif self:check('"') then return self:parse_qstring('"')
-   else return self:take_while("[^%s;{}\"'/]") end
+   else
+      local ret = {}
+      repeat
+         table.insert(ret, self:take_while("[^%s;{}\"'/]"))
+         if self:is_eof() then break end
```

Comment:
The code looks good!  I am fine with it as-is.  Would you mind testing on a softwire-config with a big (1M) binding table and making sure parsing performance doesn't regress?  Thanks :)  That can be done after this goes in fwiw.  There are some speedups we could have here if that perf regresses.